src/actor.rs

```diff
@@ -109,7 +124,7 @@ impl<'a> Actor<'a> {
     pub fn deploy_remote(&self, git_location: &str, file_name: &str) -> Result<Actor, Error> {
```

Comment:
Can we change this param from git_location to remote_location?

Reply:
and done